gnomad/utils/vcf.py

```diff
@@ -1244,19 +1251,44 @@ def make_vcf_filter_dict(
             " be 'RF' or 'AS_VQSR'"
         )
 
-    filter_dict = {
-        "AC0": {
-            "Description": (
-                "Allele count is zero after filtering out low-confidence genotypes (GQ"
-                " < 20; DP < 10; and AB < 0.2 for het calls)"
+    if joint:
+        filter_dict = {
+            "PASS": {
+                "Description": "Passed variant filters in exomes and genomes, "
+                "or Passed variant filters in one of the exomes "
+                "or genomes datasets but the variant was not present in the other dataset"
+            },
+            "EXOMES_FILTERED": {
+                "Description": "Failed variant filters in exomes "
+                "dataset, but either passed variant filters in genomes dataset or the variant was not present in the genomes dataset"
+            },
+            "GENOMES_FILTERED": {
+                "Description": "Failed variant filters in genomes "
+                "dataset, but either passed variant filters in exomes dataset or the variant was not present in the exomes dataset"
+            },
+            "BOTH_FILTERED": {
+                "Description": "Failed variant filters in both exomes and genomes datasets"
+            },
+        }
+    else:
+        # The cutoffs must be specified for the filter descriptions to be generated
+        if snp_cutoff is None or indel_cutoff is None or inbreeding_cutoff is None:
+            raise ValueError(
+                "snp_cutoff, indel_cutoff, and inbreeding_cutoff must be specified to generate filter descriptions."
```

Comment:
Good catch! Let's just move it up to near the other ValueError and just add a `and not joint` to the condition